examples/gallery/plot/points-categorical.py

```diff
@@ -0,0 +1,73 @@
+"""
+Color points by categories
+---------------------------
+The :meth:`pygmt.Figure.plot` method can be used to plot symbols which are
+color-coded by categories. In the example below, we show how the
+`palmerpenguins dataset <https://github.com/allisonhorst/palmerpenguins>`__
+can be visualized. Here, we can pass the individual categories included in
+the species column directly to the ``color`` parameter via
+``color=df.species.cat.codes.astype(int)``. Additionally, we have to set
+``cmap=True``. A desired colormap can be selected via the :meth:`pygmt.makecpt`
+method.
+"""
+
+import numpy as np
+import pandas as pd
+import pygmt
+
+# Load sample penguins data and convert 'species' column to categorical dtype
+df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/penguins.csv")
+df["species"] = df.species.astype(dtype="category")
```

Comment:
It makes no differences, but more consitent throughout the whole code.
```suggestion
df.species = df.species.astype(dtype="category")
```